File: haskell/app/MusicPlayer.hs
module MusicPlayer where

import System.Process (callCommand)

tocarMusica :: Int -> IO ()
tocarMusica m = do
    if  m == 1 then callCommand "cvlc --play-and-exit haskell/songs/Even.mp3 > /dev/null 2>&1  &"
    else if m == 2 then callCommand "cvlc --play-and-exit haskell/songs/Paint.mp3 > /dev/null 2>&1  &"
    else callCommand "cvlc --play-and-exit haskell/songs/Bulls.mp3 > /dev/null 2>&1  &"
    
tocarMusicaMenu :: IO ()
tocarMusicaMenu = do
    callCommand "cvlc --loop haskell/songs/Menu.mp3 > /dev/null 2>&1 &"
    
pararMusica :: IO ()
pararMusica = callCommand "pkill vlc"

File: prolog/MusicPlayer.pl
:- module(musicPlayer, [tocarMusica/1, tocarMusicaMenu/0, pararMusica/0]).

tocarMusica(Musica) :-
    (Musica = "1" -> shell("cvlc --play-and-exit musicas/Even.mp3 > /dev/null 2>&1  &");
    Musica = "2" -> shell("cvlc --play-and-exit musicas/Paint.mp3 > /dev/null 2>&1  &");
    shell("cvlc --play-and-exit musicas/Bulls.mp3 > /dev/null 2>&1  &")).

tocarMusicaMenu :-
    shell("cvlc --loop musicas/Menu.mp3 > /dev/null 2>&1 &").

pararMusica :-
    shell("pkill vlc").